### ISSUE TITLE
vscode extension can not spawn electron process after package to vsix

### ISSUE THREAD

**Issue by njleonzhang** [Microsoft/vscode#55631]
- VSCode Version: 1.25.1 (1.25.1)
- OS Version:  mac os 10.13.6

Steps to Reproduce:

1.  clone the project https://github.com/njleonzhang/vscode-extension-mardown-image-paste
2. install the dependency via `npm install` 
3. open it in vscode.
4. run in test mode, press `F5`
5. run plugin command in the `extension development vscode instance`: `Paste Image`.
6. A electron app should be opened.
7. build the extension to vsix via `npm run package`
8. install the vsix to vscode.
9.  run plugin command: `Paste Image`.

expected:
   A electron app should be opened.

fact： 
   no electron app is opened.

# About the extension
It spawn electron process for leverage the electron api `clipboard.readImage` for reading image from the clipboard. It now works as expected in debug mode, but fails in release mode because `electron` can not be started. No error is shown, I do not know how to debug further. Please help.

**Comment by njleonzhang** [comment]
@isidorn it should be a bug, not a question. it is not work in release mode....

**Comment by isidorn** [comment]
fyi @joaomoreno to decide

**Comment by joaomoreno** [comment]
Yeah this is very likely an environment variable which messes Electron up.
